### ISSUE TITLE
CSV files containing carriage returns are not parsed properly.

### ISSUE THREAD

**Issue by workergnome** [processing/p5.js#545]
When loading the following CSV file:

```csv
title,special,special_title,width,height,area,creation,acquisition
"The Studio
Gift of Mr. and Mrs. Charles Zadok",0,57.26,25.000,30.750,768.75,1951,1957
```

The first entry of the second line (the *title*) should be interpreted as `The Studio\nGift of Mr. and Mrs. Charles Zadok`.  Instead, it is loading it as a complete record with a single entry, `"The Studio`, and then assuming there is a second row, with the title of `Gift of Mr. and Mrs. Charles Zadok"`.

According to [RFC 4180](http://tools.ietf.org/html/rfc4180), new lines within quotes should be considered part of the data, not record delimiters.

**Comment by workergnome** [comment]
And I'm an idiot who didn't read the documentation fully.

**Comment by workergnome** [comment]
Actually, reading the documentation isn't helpful, either.  The documentation mentions the newlines flag, but the code doesn't actually implement it.

**Comment by workergnome** [comment]
Fixed in #551 .
